clients/python/README.md

```diff
@@ -105,12 +104,12 @@ Here some examples of **BAD** bad choices for `source_id`:
 ### Advanced configuration
 
 Samsara SDK buffers events and peridically flushes the events to the
-Samsara API. A ring buffer is used for this purpose. The events are
+Samsara API. `collections.deque` is used for this purpose. The events are
```

Comment:
could you put something like:

> A ring buffer (`collection.deque`) is used ...

the deque has more similarities with a queue than with a ring-buffer. But the particular python implementation can be used in place of a ring-buffer